zebra-script/src/lib.rs

```diff
@@ -327,6 +379,35 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn get_transparent_output_address() -> Result<()> {
+        zebra_test::init();
+
+        let transaction =
+            SCRIPT_TX.zcash_deserialize_into::<Arc<zebra_chain::transaction::Transaction>>()?;
+
+        // Hashes were extracted from the transaction (parsed with zebra-chain,
```

Comment:
Looks great, thanks for the tests!

When we test the RPCs, we can add a test vector from the first 10 blocks.